### ISSUE TITLE
`rke config --system-images` prints comment line to stdout

### ISSUE THREAD

**Issue by axeal** [rancher/rancher#20781]
**What kind of request is this (question/bug/enhancement/feature request):**

Enhancement

**Steps to reproduce (least amount of steps as possible):**

`rke config --system-images` prints both the list of images, one-per-line, and an initial info level log message to stdout.

As a result, if this output is redirected to file for consumption by the airgapped install helper scripts `rancher-save-image.sh` or `rancher-load-images.sh`, it cannot be parsed successfully by these scripts without removing the log line. Failure to remove this results in an error, per the following:

```
./rke config --system-images >> ./rancher-images.txt
./rancher-save-images.sh --image-list ./rancher-images.txt
++ cat ./rancher-images.txt.old
+ for i in $(cat ${list})
+ docker pull 'time="2019-06-11T10:54:35Z"'
invalid reference format: repository name must be lowercase
```

Could the print of the log line be removed, or a flag added to prevent printing it?

**Environment information**
RKE v0.2.4

**Comment by dnoland1** [comment]
I'd consider this a bug. In the docs at https://rancher.com/docs/rancher/v2.x/en/installation/air-gap-high-availability/prepare-private-registry/ , it instructs user to run `rke config --system-images >> ./rancher-images.txt` then step 6 fails:
```
./rancher-save-images.sh --image-list ./rancher-images.txt

error message:
+ for i in '$(cat ${list})'
+ docker pull 'time="2019-07-11T23:08:17Z"'
invalid reference format: repository name must be lowercase
```

**Comment by luthermonson** [comment]
PR to change to Debugf https://github.com/rancher/rke/pull/1582

**Comment by cjellick** [comment]
@galal-hussein saw the fix after it was merged and did not like it. @luthermonson is going to do a new fix

**Comment by luthermonson** [comment]
New PR to harden the save and load shell scripts: https://github.com/rancher/rancher/pull/22395
New PR to turn back on the Infof in rke and add a --list-versions param for the integration tests to use: https://github.com/rancher/rke/pull/1590

**Comment by luthermonson** [comment]
Docs update: https://github.com/rancher/docs/pull/1692

**Comment by davidnuzik** [comment]
Version: v0.3.0-rc8

I can now confirm when I execute `rke config --system-images` we are no longer getting any comment lines to stdout.

Example shows stdout when execute `./rke_linux-amd64 config --system-images` on my box:
```
WARN[0000] This is not an officially supported version (v0.3.0-rc8) of RKE. Please download the latest official release at https://github.com/rancher/rke/releases/latest
INFO[0000] Generating images list for version [v1.15.3-rancher2-1]:
rancher/coreos-etcd:v3.3.10-rancher1
rancher/rke-tools:v0.1.44
rancher/k8s-dns-kube-dns:1.15.0
rancher/k8s-dns-dnsmasq-nanny:1.15.0
rancher/k8s-dns-sidecar:1.15.0
rancher/cluster-proportional-autoscaler:1.3.0
rancher/coredns-coredns:1.3.1
rancher/hyperkube:v1.15.3-rancher2
rancher/coreos-flannel:v0.11.0-rancher1
rancher/flannel-cni:v0.3.0-rancher5
rancher/calico-node:v3.7.4
rancher/calico-cni:v3.7.4
rancher/calico-kube-controllers:v3.7.4
rancher/calico-ctl:v2.0.0
rancher/coreos-flannel:v0.11.0
weaveworks/weave-kube:2.5.2
weaveworks/weave-npc:2.5.2
rancher/pause:3.1
rancher/nginx-ingress-controller:nginx-0.25.1-rancher1
rancher/nginx-ingress-controller-defaultbackend:1.5-rancher1
rancher/metrics-server:v0.3.3
rancher/kubelet-pause:v0.1.3
```

However, one of the qualifiers for the issue was that we don't show the [INFO] line in our stdout and I still see this (The WARN message is ONLY because this is not an official release yet, this will go away in a release build). If I were to pipe/redirect this that [INFO] line would be sent in stdout...

I am reopening the issue due to this.

**Comment by davidnuzik** [comment]
Maybe the pr didn't make it in this rc from 2 days ago. investigating...

**Comment by cjellick** [comment]
@luthermonson please ellaborate on how to test. @davidnuzik tl;dr - theres a --quiet flag

**Comment by luthermonson** [comment]
@davidnuzik youneed to add "quiet" like this `./rke_linux-amd64 --quiet config --system-images`

**Comment by davidnuzik** [comment]
Ah okay thanks my bad for not catching that (docs issue)

**Comment by cjellick** [comment]
This needs tested in the context of @luthermonson 's pending doc changes for air gap install. Will let him give the details on that

**Comment by luthermonson** [comment]
@davidnuzik the context for no output is from this doc change: https://github.com/rancher/docs/pull/1692

**Comment by davidnuzik** [comment]
If we need to test end-to-end (test airgap) then @jiaqiluo may need to take the issue. cc: @sangeethah 
If me testing the new quiet flag and generating the images text files and validating that output is good is not sufficient.

**Comment by sangeethah** [comment]
@jiaqiluo Could you check on the related doc changes  during the next attempt of airgap installs ?

**Comment by jiaqiluo** [comment]
@sangeethah  Sure!

The bug fix and the doc changes will be validated during the next attempt of airgap installs.

**Comment by jiaqiluo** [comment]
The bug fix is validated with rke version v0.3.0-rc9

```
> rke --v
rke version v0.3.0-rc9
> rke --quiet config --system-images >> ./rancher-images.txt 
```
 
There is no extra timestamp line in the list of images
